containers/dapr-dotnetcore-3.1/.devcontainer/library-scripts/common-debian.sh

```diff
@@ -0,0 +1,340 @@
+#!/usr/bin/env bash
```

Comment:
Since you're using the pre-built image, you don't need common-debian.sh or node-debian.sh - these are in the .NET / Node definitions for use with `base.Dockerfile.`

Reply:
Not sure why I thought it was necessary; removed.